sos/report/plugins/scsi.py

```diff
@@ -33,6 +33,11 @@ def setup(self):
         self.add_cmd_output("lspath")
         self.add_cmd_output("lsmap -all")
         self.add_cmd_output("lsnports")
+        self.add_cmd_output("lsscsi -H")
+        self.add_cmd_output("lsscsi -g")
+        self.add_cmd_output("lsscsi -d")
+        self.add_cmd_output("lsscsi -s")
+        self.add_cmd_output("lsscsi -L")
```

Comment:
Let's do this as a single call to `add_cmd_output()` with a list instead.

Reply:
ok

Reply:
It looks like these got dropped in the most recent branch update, was that intentional?